### DISCUSSION TITLE
Disablling browser history state on a component?

### DISCUSSION THREAD

**Discussion by andrew-ireland** [livewire/livewire#5091] [1. Help]
Please forgive any ignorance on my part, i have limited experience with livewire/laravel, but have managed to scrape together the following page/component as a test-case.

Still a bit rough around the edges, but this is it: [https://staging.mass.net.au/projects](https://staging.mass.net.au/projects)

I'm not sure livewire is best solution here, as I'm relying on a fair bit of javascript on the page... but it seems to be working pretty well overall! 

The issue I'm faced with now though, is the back/forward browser buttons break the whole illusion of the experience...  I.e. click a few filter buttons, navigate to another page and then try to navigate back—it's pretty ugly! Essentially the history navigation breaks 'the illusion' of the page. None of the javascript hooks the page relies on fire properly, and the user is forced to cycle back through every filter step, and every minor 'load more' request I have in place using intersectionObserver. While navigating through this the page visually looks broken, as certain javascript events aren't currently firing.

Best case scenario for UX imo would be to trash the 'granular' history on the livewire component if I can, so the dynamic livewire interactions aren't stored as separate history events.... does that make sense? That's what I'm hoping to work out if I can...

TLDR; I've love it if I could have my livewire component not send browser history events entirely, or stop livewire from tracking the back/forwards state completely — is that possible? Is there a way to disable that entirely? Alternatively, anyone have any ideas how I can improve the browser history experience here? I'm at a loss currently 🥲

**Comment by joshhanley** [comment]
@andrew-ireland hard to say without seeing your code, but it looks like you are using `queryString` for category. If you remove than, then the history shouldn't be stored for the changed between categories (for example).

Hope this helps!
